doc/tgz.md

```diff
@@ -0,0 +1,57 @@
+# Managing TGZ package
+
+Deploying TGZ archive is different from deploying RPM or DEB.
+It requires additional actions such as configuring systemd units.
+All these actions can be disabled.
+
+Using TGZ package allows to use [multiversion approach](./multiversion.md)
+that can be helpful to perform rolling update correclty.
+This approach uses links to specify version of application that is used by each
+instance.
+
+## Unpacking TGZ
+
+TGZ package contains one directory named `<app-name>` that contains application.
+By default, it is unpacked to `cartridge_install_dir` directory according to
+[multiversion](./multiversion.md) policy.
+
+### Installing Tarantool
+
+If package doesn't contain `tarantool` binary, opensource Tarantool is installed.
+It's version is described in `VERSION` file that is created by Cartridge CLI
+on application packing.
+
+Use `cartridge_install_tarantool_for_tgz` flag to disable installation
+of Tarantool.
+
+### Application user ang group
+
+Application user that owns all application files and instances processes
+can be configured using `cartridge_app_user` and `cartridge_app_group` variables.
+
+### Application directories
+
+Application directories are created before unpacking.
+To configure them use
+
+* `cartridge_data_dir`
+* `cartridge_run_dir`
+* `cartridge_conf_dir`
+* `cartridge_install_dir`
+* `cartridge_instances_dir` in case of using [multiversion](./multiversion.md)
```

Comment:
What about to describe it?

Reply:
I think that it's good to keep variables description in one place (in multiversion doc in this case) and use links in other places.

Reply:
What about to make link like `description of variables [here](/doc/multiversion.md)`?